CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 IMPROVEMENTS: 
 
+* Use consul-ca-cert for sync-catalog when using autoencrypt and client is disabled. [[GH-891](https://github.com/hashicorp/consul-helm/pull/891)]
```

Comment:
I think this is a bugfix since we did allow sync catalog without clients before, it just would break in a certain configuration. I'd say something like "Sync Catalog: fix issue running with clients disabled and auto encrypt enabled"